aip/general/0008.md

```diff
@@ -114,18 +114,29 @@ followed by a bulleted list explaining the example.
 Individual subsections can be cited individually, and further elaborate
 details.
 
+## Rationale
+
+The "rationale" section is optional, and helps the reader understand the
+motivation for specific guidance within the AIP.
+
+Deeper explanations of design justification and tradeoffs **must** be in the
+rationale instead of other sections, to ensure the rest of the document acts as
+an easily digestible reference.
+
+## History
+
+The "history" section is optional, and documents events and context around
+a significant edit (e.g. rewrite) of an AIP.
+
+- The section **must not** be used to exhaustively enumerate all changes (this
```

Comment:
Should this be bulleted?

Reply:
i went back and forth, but thought bullets might be a good way to express it, in case there's other must/shoulds/etc.

stylistically I feel like we should always have one-liner bullets for M/S/M-not/S-not.

Reply:
What's the difference between history and changelog?

Reply:
history is for more free-form text on significant events, while changelog is *any* change.

But of course, that fact that you're asking this question implies I need to clarify. I'll add a sentence.

Reply:
+1 no bullets. Where's the other bullet(s)? It is weird to have a list with one bullet. In fact, it is grammatically incorrect, if we're going to be draconian about it.

Reply:
okay, bullet removed.